### PR TITLE
ref(app): Consolidate runtime publicpath setup logic

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Hopefully this makes it a bit more clear what's going on here

### CODE REVIEW

File: src/sentry/static/sentry/app/publicPath.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint no-native-reassign:0 */
+/* global __webpack_public_path__ */
+
+/**
+ * Set the webpack public path at runtime. The __sentryGlobalStaticPrefix will
+ * be declared in layout.html.
+ *
+ * NOTE: This MUST be loaded before any other modules in the entrypoint.
```
Comment (markstory) [inline src/sentry/static/sentry/app/publicPath.jsx]: But its after the bootstrap imports.

File: src/sentry/static/sentry/app/index.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import 'bootstrap/js/alert';
 import 'bootstrap/js/tab';
 import 'bootstrap/js/dropdown';
 
+import 'app/publicPath';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/index.js]: can we move this to utils like `emotion-setup` below

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/index.js]: Yeah sounds good

File: src/sentry/static/sentry/app/publicPath.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/publicPath.jsx]: Updated the comment. It can be after bootstrap, just needs to be before other app modules load.

**Review by billyvg — APPROVED** [review]

**Pull request merged**